src/com/haskforce/refactoring/HaskellRefactoringSupportProvider.java

```diff
@@ -13,4 +13,6 @@
     public boolean isMemberInplaceRenameAvailable(@NotNull PsiElement element, PsiElement context) {
         return element instanceof HaskellNamedElement;
     }
+
+
```

Comment:
Done, likely happened while I was feeling my way around the classes to see what needed to be changed where.